doc/engineering-reference/src/simulation-models-encyclopedic-reference-004/system-availability-managers.tex

```diff
@@ -30,31 +30,55 @@ \subsection{Night Cycle}\label{night-cycle}
 
 The input object AvailabilityManager:NightCycle is to specify when a system that has been scheduled off during unoccupied hours should cycle on to prevent building temperatures from becoming too hot or too cold. This manager can not be used to force a system off. The inputs are:
 
-·~~~~~~~~A manager applicability schedule;
+\begin{itemize}
```

Comment:
Even this dog can learn some new LaTeX tricks!  And did you notice that [Building Simulation 2017](http://www.bs2017.org/instructions.html) accepts abstracts in LaTeX? Scroll down and there's a LaTeX abstract template.